src/index.html

```diff
@@ -10,20 +10,18 @@
 		<!--- TODO : <meta name="description" content="" />-->
 
 		<!-- Facebook Meta Tags -->
-		<!-- TODO: <meta property="og:url" content="" />-->
+		<meta property="og:title" content="Antiplag">
+		<meta property="og:description" content="Check your files before submitting.">
+		<meta property="og:url" content="https://antiplag.sk/">		
```

Comment:
What about extracting this value to the .env file?

Reply:
I don't understand, what I should do